polkadot-parachain/src/cli.rs

```diff
@@ -64,12 +64,22 @@ pub enum Subcommand {
 	TryRuntime,
 }
 
+const AFTER_HELP_EXAMPLE: &str = color_print::cstr!(
+	r#"<bold><underline>Examples:</></>
+   <bold>polkadot-parachain --chain statemint --sync warp -- --chain polkadot</>
```

Comment:
```suggestion
   <bold>polkadot-parachain --chain statemint --sync warp -- --chain polkadot --sync warp</>
```

You also want to warp sync the relay chain node.